container/container.go

```diff
@@ -109,6 +109,7 @@ type Container struct {
 
 	// Fields here are specific to Windows
 	NetworkSharedContainerID string
+	SharedEndpointList       []string
```

Comment:
Because these are populated only for startup (IIRC) and not generally needed across daemon restarts.